docker-compose.yml

```diff
@@ -11,7 +11,7 @@ networks:
 services:
 
   bootstrap:
-    image: ${NWAKU_IMAGE}
+    image: ${NWAKU_IMAGE:-wakuorg/nwaku:latest}
```

Comment:
+1 for using default values
but not sure I like using :latest tag, since it lacks determinism. at different points in time, this can point to a different image.

can we set to eg latest v0.xx release?

Reply:
I think that if we set it to a release, we would need to update it regularly and probably won't happen. If someone uses it, it would most likely run some old release and that might be probably worse than the issue of latest version changing constantly.

If it's ok I would leave it like this and if the non-determinism becomes an issue we change it and start using a specific release as default :)